Client/Core/Commands/MiscHandler.cs

```diff
@@ -20,11 +20,11 @@ public static partial class CommandHandler
 
             new Thread(() =>
             {
-                string tempFile = FileHelper.GetTempFilePath(".exe");
+                var tempFile = FileHelper.GetTempFilePath(".exe");
```

Comment:
Ok different projects different styles :) as an example we at work use only var :)